apps/desktop/scripts/dev.mjs

```
@@ -0,0 +1,129 @@
+// load .ts entry points at runtime.
+function awaitProc(child, label) {
+  return new Promise((resolve, reject) => {
+    child.on("error", reject);
+    child.on("exit", (code) =>
+      code === 0 ? resolve() : reject(new Error(`${label} failed (${code})`)),
+    );
+  });
+}
+
+await awaitProc(
+  spawn("bun", ["run", "--filter", "@dashframe/server-core", "build"], {
```

ESLint

[error] Make sure the "PATH" variable only contains fixed, unwriteable directories.

(sonarjs/no-os-command-from-path)